test/src/unit-lru_cache.cc

```diff
@@ -224,3 +225,35 @@ TEST_CASE_METHOD(
   auto it_end = lru_cache_->item_iter_end();
   CHECK(it == it_end);
 }
+
+TEST_CASE("BufferLRUCache of 0 capacity", "[lru_cache]") {
+  // Create 0 size cache
+  BufferLRUCache* lru_cache_ = new BufferLRUCache(CACHE_ZERO_SIZE);
```

Comment:
We typically use trailing `_` to signify a member variable. In the other unit tests in this file, the `lru_cache_` is a member of the `struct BufferLRUCacheFx`.

I'd suggest reusing the `BufferLRUCacheFx` like the other test with `TEST_CASE(BufferLRUCacheFx, ...`. If you don't want to do that, just rename this to `lru_cache`.

Reply:
I fixed the naming for now, as I am not sure how to reuse BufferLRUCacheFx here as it's using CACHE_SIZE in the constructor and I need to use 0. Anything I am missing or any ideas? I can always create another fixture if you think it's better.

Reply:
Good point. I think this looks good as is.